packages/manager/apps/adp/index.js

```diff
@@ -1,9 +1,12 @@
 import 'script-loader!jquery'; // eslint-disable-line
 import 'script-loader!lodash'; // eslint-disable-line
+import 'ovh-ui-kit/dist/oui.css';
```

Comment:
ovh-ui-kit css is being imported here because, importing this in the App causes overriding of already overridden classes by Manager, when we integrate ADP with Manager Cloud.